jquery.flot.hover.js

```diff
@@ -0,0 +1,303 @@
+/* global jQuery */
+
+(function($) {
+    'use strict';
+
+    var options = {
+            grid: {
+                hoverable: true,
+                clickable: true
+            }
+        },
+        plot, highlights = [];
```

Comment:
These variables are shared among all plots. The right pattern is to move the 'private' variables and everyone using them inside the **init**.